godot_ap/autoloads/archipelago.gd

```diff
@@ -17,6 +17,13 @@ class_name AP extends Node
 @export var AP_HIDE_NONLOCAL_ITEMSENDS := true
 ## Automatically opens a default AP text console.
 @export var AP_AUTO_OPEN_CONSOLE := false
+## Show items that are both progression and useful with their own color
+@export var AP_ENABLE_PROGUSEFUL := false:
+	set(val):
+		if val != AP_ENABLE_PROGUSEFUL:
+			AP_ENABLE_PROGUSEFUL = val
+			STATIC_AP_ENABLE_PROGUSEFUL = val
+static var STATIC_AP_ENABLE_PROGUSEFUL := false
```

Comment:
Why do you need a static? Archipelago is an autoload so you can just `Archipelago.AP_ENABLE_PROGUSEFUL`...

Reply:
because can't use non-static inside a static function

Reply:
yes you can, just not of the same class directly. But `Archipelago` is an autoload. So just use `Archipelaogo.` before the variable.

Reply:
nvm, fixed